<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Imagenes de Proyecto/desktop.png</normaloff>../Imagenes de Proyecto/desktop.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Imagenes de Proyecto/desktop.png</normaloff>../Imagenes de Proyecto/desktop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Imagenes de Proyecto/buscar.png</normaloff>../Imagenes de Proyecto/buscar.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>160</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>581</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>581</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QListWidget" name="listWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>581</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <widget class="QMenu" name="menuNuevo_Archivo">
     <property name="title">
      <string>Nuevo Archivo</string>
     </property>
     <addaction name="actionArchivo_de_Texto"/>
     <addaction name="actionMapa_de_bits"/>
     <addaction name="actionArchivo_Winrar"/>
    </widget>
    <addaction name="actionNueva_Carpeta"/>
    <addaction name="menuNuevo_Archivo"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionCopiar"/>
    <addaction name="actionCortar"/>
    <addaction name="actionPegar"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAtras"/>
   <addaction name="actionAdelante"/>
   <addaction name="separator"/>
   <addaction name="actionArchivo_Winrar_2"/>
   <addaction name="actionMapa_de_Bit"/>
   <addaction name="actionNuevo_Archivo_2"/>
   <addaction name="separator"/>
   <addaction name="actionNueva_Carpeta_2"/>
   <addaction name="separator"/>
   <addaction name="actionCopiar_2"/>
   <addaction name="actionCortar_2"/>
   <addaction name="actionPegar_2"/>
   <addaction name="actionEliminar"/>
   <addaction name="separator"/>
   <addaction name="actionIconos_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNueva_Carpeta">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/Folder-Add-icon.png</normaloff>../Imagenes de Proyecto/Folder-Add-icon.png</iconset>
   </property>
   <property name="text">
    <string>Nueva Carpeta</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/Salir.png</normaloff>../Imagenes de Proyecto/Salir.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/copiar.png</normaloff>../Imagenes de Proyecto/copiar.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/Cortar.png</normaloff>../Imagenes de Proyecto/Cortar.png</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/pegar.png</normaloff>../Imagenes de Proyecto/pegar.png</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
  <action name="actionLista">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/lista.png</normaloff>../Imagenes de Proyecto/lista.png</iconset>
   </property>
   <property name="text">
    <string>Lista</string>
   </property>
  </action>
  <action name="actionIconos">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/icono.png</normaloff>../Imagenes de Proyecto/icono.png</iconset>
   </property>
   <property name="text">
    <string>Iconos</string>
   </property>
  </action>
  <action name="actionDetalles">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/view_grid-512.png</normaloff>../Imagenes de Proyecto/view_grid-512.png</iconset>
   </property>
   <property name="text">
    <string>Detalles</string>
   </property>
  </action>
  <action name="actionAtras">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/Atras.png</normaloff>../Imagenes de Proyecto/Atras.png</iconset>
   </property>
   <property name="text">
    <string>atras</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Atras&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionAdelante">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/Adelante.png</normaloff>../Imagenes de Proyecto/Adelante.png</iconset>
   </property>
   <property name="text">
    <string>adelante</string>
   </property>
   <property name="toolTip">
    <string>adelante</string>
   </property>
  </action>
  <action name="actionNuevo_Archivo_2">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/archivo.png</normaloff>../Imagenes de Proyecto/archivo.png</iconset>
   </property>
   <property name="text">
    <string>Archivo Texto</string>
   </property>
   <property name="toolTip">
    <string>Nuevo Archivo</string>
   </property>
  </action>
  <action name="actionNueva_Carpeta_2">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/Folder-Add-icon.png</normaloff>../Imagenes de Proyecto/Folder-Add-icon.png</iconset>
   </property>
   <property name="text">
    <string>Nueva Carpeta</string>
   </property>
   <property name="toolTip">
    <string> Nueva Carpeta</string>
   </property>
  </action>
  <action name="actionCopiar_2">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/copiar.png</normaloff>../Imagenes de Proyecto/copiar.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="toolTip">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionCortar_2">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/Cortar.png</normaloff>../Imagenes de Proyecto/Cortar.png</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
   <property name="toolTip">
    <string>Cortar</string>
   </property>
  </action>
  <action name="actionPegar_2">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/pegar.png</normaloff>../Imagenes de Proyecto/pegar.png</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
   <property name="toolTip">
    <string>Pegar</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/eliminar.png</normaloff>../Imagenes de Proyecto/eliminar.png</iconset>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="toolTip">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="actionArchivo_de_Texto">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/txt.png</normaloff>../Imagenes de Proyecto/txt.png</iconset>
   </property>
   <property name="text">
    <string>Archivo de Texto</string>
   </property>
  </action>
  <action name="actionMapa_de_bits">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/picture.png</normaloff>../Imagenes de Proyecto/picture.png</iconset>
   </property>
   <property name="text">
    <string>Mapa de bits</string>
   </property>
  </action>
  <action name="actionArchivo_Winrar">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/WinRAR.png</normaloff>../Imagenes de Proyecto/WinRAR.png</iconset>
   </property>
   <property name="text">
    <string>Archivo Winrar</string>
   </property>
  </action>
  <action name="actionMapa_de_Bit">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/picture.png</normaloff>../Imagenes de Proyecto/picture.png</iconset>
   </property>
   <property name="text">
    <string>Mapa de Bit</string>
   </property>
   <property name="toolTip">
    <string>Mapabit</string>
   </property>
  </action>
  <action name="actionArchivo_Winrar_2">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/WinRAR.png</normaloff>../Imagenes de Proyecto/WinRAR.png</iconset>
   </property>
   <property name="text">
    <string>Archivo Winrar</string>
   </property>
   <property name="toolTip">
    <string>Winrar</string>
   </property>
  </action>
  <action name="actionIconos_2">
   <property name="icon">
    <iconset>
     <normaloff>../Imagenes de Proyecto/icono.png</normaloff>../Imagenes de Proyecto/icono.png</iconset>
   </property>
   <property name="text">
    <string>Vista</string>
   </property>
   <property name="toolTip">
    <string>Vista</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
